Detectors/TPC/reconstruction/include/TPCReconstruction/BoxClusterer.h

```diff
@@ -57,8 +57,8 @@ namespace o2{
       /// @param mcDigitTruth MC Digit Truth container
       /// @param eventCount event counter 
       /// @return Container with clusters
-      void Process(std::vector<o2::TPC::Digit> const & digits, MCLabelContainer const* mcDigitTruth, int eventCount) override;
-      void Process(std::vector<std::unique_ptr<Digit>>& digits, MCLabelContainer const* mcDigitTruth, int eventCount) override;
+      void Process(std::vector<o2::TPC::Digit> const* digits, MCLabelContainer const* mcDigitTruth, int eventCount) override;
+      void Process(std::vector<std::unique_ptr<Digit>>* digits, MCLabelContainer const* mcDigitTruth, int eventCount) override;
```

Comment:
why do you change to a `pointer of vectors`? If the digits are just consumed, consider using `std::vector<> const&`

Reply:
I would rather delete this class completely since no one is really maintaining it (I made those changes probably earlier in the way of removing all plain pointers). I think it is not even working with the continuous readout (see question to Jens)

Reply:
For the moment I would be in favour of keeping the BoxClusterer, since it is a very trivial algorithm.
I agree with Sandro it should be `const &` as for the HwClusterer.